lib/components/map/point-popup.tsx

```diff
@@ -56,22 +79,31 @@ function MapPopup({
       closeOnClick
       latitude={mapPopupLocation.lat}
       longitude={mapPopupLocation.lon}
-      onClose={() => setMapPopupLocation({ location: null })}
+      onClose={clearMapPopupLocation}
+      // Override inline style supplied by react-map-gl to accommodate long "plan a trip" translations.
+      style={{ maxWidth: '260px', width: '260px' }}
     >
-      <PopupContainer>
-        <PopupTitle>
-          {typeof popupName === 'string' && popupName.split(',').length > 3
-            ? popupName.split(',').splice(0, 3).join(',')
-            : popupName}
-        </PopupTitle>
-        <div>
-          <FromToLocationPicker
-            label
-            location={{ ...mapPopupLocation, name: popupName }}
-            setLocation={onSetLocationFromPopup}
-          />
-        </div>
-      </PopupContainer>
+      <ZoomButton
+        className="pull-right"
```

Comment:
I wouldn't use float for this. We can `flex` the container instead. That way we can `align-items: center/baseline` as well

Reply:
Did you intend the magnifier to center vertically when the place name occupies multiple lines? My intuition was rather to leave it at a fixed place relative to the top-right corner of the popup, similar with the "close" button elsewhere.

Reply:
Off the top of my head not sure if it would look better to align it with the first text line or the entire text block but I definitely think it should be tied to the text in some way -- the magnifying glass has nothing to do with the popup, but rather the title. The close button should be semantically tied to the popup as the close button affects the popup and only has to do with the popup. The zoom button's context is firmly tied to the active stop though.

Reply:
I switched to `flex` and improved the alignment of the magnifier with the top line of the place text with https://github.com/opentripplanner/otp-react-redux/pull/667/commits/3e590c94392954fd6870e1895e5205d82b72a9f1.